HEURISTIC:
  ant_cycles:   50
  whale_num:    100
  whale_cycles: 60
  ALim:         1
  beta:         2
